drivers/usb/host/dwc_otg/dwc_otg_hcd.c

```diff
@@ -198,8 +199,17 @@ static void kill_urbs_in_qh_list(dwc_otg_hcd_t * hcd, dwc_list_link_t * qh_list)
 				qh->channel->halt_status = DWC_OTG_HC_XFER_URB_DEQUEUE;
 				qh->channel->halt_pending = 1;
 				if (hcd->fiq_state->channel[n].fsm == FIQ_HS_ISOC_TURBO ||
-					hcd->fiq_state->channel[n].fsm == FIQ_HS_ISOC_SLEEPING)
+				    hcd->fiq_state->channel[n].fsm == FIQ_HS_ISOC_SLEEPING)
```

Comment:
wrong whitespace?

Reply:
No, I think the use of spaces is allowed for parameter indentation provided they appear after any tabs.

Reply:
That was what I was going for - fixing parameter indents. Multi-line if() statements should align with the open bracket using spaces after the last tab stop.